docs/typescriptlang/zh/index2.ts

```diff
@@ -1,79 +1,79 @@
 export const indexCopy = {
   index_2_headline:
-    "TypeScript is <bold>JavaScript with syntax for types.</bold>",
-  index_2_byline: "TypeScript extends JavaScript by adding types.",
+    "TypeScript 是<bold>带有类型的 JavaScript</bold>",
+  index_2_byline: "TypeScript 通过添加类型扩展了 JavaScript",
   index_2_summary:
-    "TypeScript is a strongly typed programming language that builds on JavaScript, giving you better tooling at any scale.",
+    "TypeScript 是一种基于 JavaScript 的强类型编程语言，在任意规模的应用上都是更有力的工具。",
 
   // Above the fold
-  index_2_cta_install: "Try TypeScript Now",
-  index_2_cta_install_subtitle: "Online or via npm",
+  index_2_cta_install: "现在就来试试 TypeScript",
+  index_2_cta_install_subtitle: "在线尝试或者使用 npm",
 
-  index_2_cta_play: "In your browser",
-  index_2_cta_play_subtitle: "via the Playground",
+  index_2_cta_play: "在浏览器中尝试",
+  index_2_cta_play_subtitle: "使用演练场",
 
-  index_2_cta_download: "On your computer",
-  index_2_cta_download_subtitle: "via npm",
+  index_2_cta_download: "本地尝试",
+  index_2_cta_download_subtitle: "使用 npm",
 
   // Editor titles
-  index_2_tab_1: "Editor Checks",
-  index_2_tab_2: "Auto-complete",
-  index_2_tab_3: "Interfaces",
+  index_2_tab_1: "编辑器检查",
+  index_2_tab_2: "自动补全",
+  index_2_tab_3: "接口",
   index_2_tab_4: "JSX",
   index_2_tab_5: "ESNext",
 
   // Quick pitch
-  index_2_what_is: "What is TypeScript?",
-  index_2_what_is_js: "JavaScript and More",
+  index_2_what_is: "什么是 TypeScript？",
+  index_2_what_is_js: "比 JavaScript 更多",
```

Comment:
```suggestion
  index_2_what_is_js: "TypeScript 是 JavaScript 的超集",
```

Or this more direct one?